front/form/form.form.php

```diff
@@ -43,17 +43,25 @@
 // Read parameters
 $id = $_REQUEST['id'] ?? null;
 
-if (isset($_POST["add"])) {
-    // Create form
-    $form = new Form();
-    $form->check($id, CREATE);
+if (($_GET['id'] ?? 0) == 0) {
+    // Clear stale drafts
+    // TODO: move to a dedicated cron task
+    $DB->delete(Form::getTable(), [
```

Comment:
You should use the `Form::delete()` method. Maybe plugins will want to plug some event handler on draft deletion process.